x11-themes/sugar-candy/sugar-candy-1.6.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Sugar Candy theme for SDDM"
+HOMEPAGE="https://www.opencode.net/marianarlt/sddm-sugar-candy"
+
+KEYWORDS="~amd64 ~x86"
+SRC_URI="https://github.com/Kangie/sddm-sugar-candy/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="svg jpeg"
+
+# Require at least one kind of image support. Kind of useless otherwise.
+REQUIRED_USE="!jpeg? ( svg )
+!svg? ( jpeg )"
+
+DEPEND=">=dev-qt/qtquickcontrols2-5.11
+svg? ( >=dev-qt/qtsvg-5.11 )
+>=dev-qt/qtgraphicaleffects-5.11
+>=dev-qt/qtdeclarative-5.11
+>=dev-qt/qtgui-5.11[jpeg?]
+>=x11-misc/sddm-0.18"
+RDEPEND="${DEPEND}"
+BDEPEND=""
+
+S="${WORKDIR}/sddm-sugar-candy-${PV}"
+
+src_install() {
```

Comment:
I guess the default Makefile or whatever does not install for you?

Reply:
Upstream installation instructions are (paraphrasing) "download a tar of the current version of the repo and extract to the sddm themes directory". I could create a makefile, but the install is trivial.